packages/components/psammead-grid/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.1.7 | [PR#3073](https://github.com/bbc/psammead/pull/3073) Update Grid Stories to include new major version of MediaIndicator |
```

Comment:
After this PR gets merged, we have to bump the `psammead-media-indicator` in the root package.json, to make sure these changes are really applied.